pyproject.toml

```diff
@@ -104,3 +104,9 @@ reverse_relative = true
 [build-system]
 requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
 build-backend = "poetry_dynamic_versioning.backend"
+
+[tool.codespell]
+skip = '.git,*.pdf,*.svg,poetry.lock,output,*.tsv,./tests/input,old'
+# some specific phrases, variables and mixed case (CamelCase etc)
+ignore-regex = '\b(Torsades de pointes|[A-Z][a-zA-Z]*|[a-z]+[A-Z][a-zA-Z]*|de pointes)\b|\bcommments:'
+ignore-words-list = 'langual,sting,infarction,holliday,cyclin,convertor,ser,collapsin,infarctions,euclidian,dependant,vrsatile'
```

Comment:
just a comment/link for myself who isn't in the domain: https://vrsatile.readthedocs.io/